src/shape/Sector.tsx

```diff
@@ -247,5 +247,5 @@ export const Sector: React.FC<Props> = sectorProps => {
     path = getSectorPath({ cx, cy, innerRadius, outerRadius, startAngle, endAngle });
   }
 
-  return <path {...filterProps(props, true)} className={layerClass} d={path} role="img" />;
+  return <path {...filterProps(props, true)} className={layerClass} d={path} />;
```

Comment:
as long as this doesn't have any negative impact, all good to me. I think we've removed these roles other places?